docs/getting_started/installation/gpu.rocm.inc.md

```diff
@@ -185,7 +186,7 @@ Building the Docker image from source is the recommended way to use vLLM with RO
     }
     ```
 
-    To build vllm on ROCm 7.0 for MI200 and MI300 series, you can use the default:
+To build vllm on ROCm 7.0, you can use the default:
```

Comment:
drop MI200 and Mi300, or alternatively specify all architectures (MI350 as well)